test/module/shared_model/backend_proto/proto_batch_test.cpp

```diff
@@ -9,49 +9,102 @@
 #include "framework/batch_helper.hpp"
 #include "framework/result_fixture.hpp"
 #include "interfaces/iroha_internal/transaction_batch.hpp"
-#include "interfaces/iroha_internal/transaction_batch_factory.hpp"
-#include "validators/field_validator.hpp"
-#include "validators/transaction_validator.hpp"
-#include "validators/transactions_collection/batch_order_validator.hpp"
+#include "interfaces/iroha_internal/transaction_batch_factory_impl.hpp"
 
 using namespace shared_model;
 using ::testing::_;
 using ::testing::Return;
 using ::testing::Test;
 using ::testing::Truly;
 
-/**
- * Creates valid unsigned transaction
- * @param created_time assigned to transactions
- * @return std::shared_ptr<interface::Transaction> containing valid unsigned
- * transaction
- */
-auto createValidUnsignedTransaction(size_t created_time = iroha::time::now()) {
-  return std::shared_ptr<interface::Transaction>(clone(
-      framework::batch::prepareTransactionBuilder("valid@account", created_time)
-          .build()));
-}
-
-/**
- * Creates invalid unsigned transaction
- * @param created_time assigned to transactions
- * @return std::shared_ptr<interface::Transaction> containing invalid unsigned
- * transaction
- */
-auto createInvalidUnsignedTransaction(
-    size_t created_time = iroha::time::now()) {
-  return std::shared_ptr<interface::Transaction>(
-      clone(framework::batch::prepareTransactionBuilder("invalid#@account",
-                                                        created_time)
-                .build()));
-}
+class TransactionBatchTest : public Test {
+ public:
+  /**
+   * Creates valid unsigned transaction
+   * @param created_time assigned to transactions
+   * @return std::shared_ptr<interface::Transaction> containing valid unsigned
```

Comment:
Type is redundant since:
1. Function is simple and it's easy to grasp
2. May lead to inconsistency in case function changes